### ISSUE TITLE
`indentation` passes over a line of a value, a selector or a set of params whose indentation opens with a bare carriage return or a form feed

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#513]
### What minimal example or steps are needed to reproduce the bug?

`indentation` measures the lines of a declaration's value, a rule's selector and an at-rule's params in `checkMultilineBit`, and finds the indentation of each line behind a break with `LEADING_INDENT_AND_CONTENT`, `/^([ \t]*)\S/`: spaces and tabs, then content. A bare carriage return and a form feed are whitespace to PostCSS and no line — so a line opening with either has content behind whitespace the pattern does not name, matches nothing, and is left out of the measurement whatever its level.

```css
a {
	b:
	1px;
}
```

With a bare carriage return in front of the tab of the third line — `a {\n\tb:\n\r\t1px;\n}` — that line draws no warning under `"tab"`, where the same file without the carriage return reports `3:2` as `Expected indentation of 2 tabs`:

| input | warnings |
| --- | --- |
| `a {\n\tb:\n\t1px;\n}` | `3:2 Expected indentation of 2 tabs` |
| `a {\n\tb:\n\r\t1px;\n}` | none |
| `a {\n\tb:\n\r\t\t1px;\n}` | none |
| `a,\n\r\tb {}` | none |
| `@media\n\r\tscreen {}` | none |

Measured on `9d0a48b` under `postcss`, `postcss-scss` and `postcss-less` alike; a form feed in the carriage return's place reads the same.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

A line is measured whatever whitespace the tokenizer read in front of its content, the way the rule reads the indentation of a node out of its `raws.before` — the whole run behind the last break — and the way the fixes of #452 write it: a carriage return or a form feed in front of the tab is part of the indentation, and `\r\t` is not one tab.

### What actually happened?

The line is passed over in silence, whether it stands at its level or not, and `--fix` leaves it as it is.

### Notes

Found while closing #452, which is the other half of the same reading: there the check read the run whole and the fix looked for spaces and tabs; here the check itself looks for spaces and tabs. The fix positions `checkMultilineBit` collects carry the measured run's length, so once the line is measured its fix lands right.
